community/neo4j/src/test/java/upgrade/StoreUpgraderInterruptionTestIT.java

```diff
@@ -129,10 +132,12 @@ public void migrate( File sourceStoreDir, File targetStoreDir, SchemaIndexProvid
         assertTrue( checkNeoStoreHasLatestVersion( check, workingDirectory ) );
         assertTrue( allStoreFilesHaveNoTrailer( fs, workingDirectory ) );
 
+        startStopDatabase( workingDirectory );
```

Comment:
This is needed for recovery, right? Shouldn't migrator leave a consistent store after doing it's job?